packages/esm-home-app/src/homepage-metrics/metrics-container.component.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import MetricTile from './metrics-tile/metrics-tile.component';
+import styles from '../homepage-metrics/home-metrics-widget.scss';
+import useAppointmentsData from '../hooks/useAppointmentsData';
+import useActiveVisits from '../hooks/useActiveVisits';
+import useVisitSummary from '../hooks/useVisitSummary';
+
+const MetricsContainer: React.FC = () => {
+  const { data: appointmentsData } = useAppointmentsData();
+  const { data: activeVisitsData } = useActiveVisits();
+  const { data: visitSummaryData } = useVisitSummary();
```

Comment:
These don't belong here. `esm-home` should just set up the slot and the logic should live in the respective esms similar to the way the appointments widget is lives in the appointments esm but is attached to the home dashboard slot